hello-react/src/App.js

```diff
@@ -0,0 +1,58 @@
+import logo from './logo.svg';
+import './App.css';
+
```

Comment:
in this case when data is added and there is no server like static pages you can easely mock some object in order to espect that from seerver later as a front end you will like it as a backend they gonna love it 

```
const dataCardArray = [
{
title: "budooo",
image: "image url.com",
text : "like pro",
}
]

```